test/qunit/model/FHIRListBinding.integration.js

```diff
@@ -8,7 +8,8 @@ sap.ui.define(["../utils/TestUtilsIntegration", "../utils/TestUtils"], function
 	}
 
 	function createListBinding(sPath, oContext, aSorters, aFilters, mParameters) {
-		oListBinding = oModel.bindList(sPath, oContext, aSorters, aFilters, mParameters).initialize();
+		oListBinding = oModel.bindList(sPath, oContext, aSorters, aFilters, mParameters);
```

Comment:
why changing these lines?

Reply:
@flovogt : I forgot to send the binding object while overwridding the method. Fixed it now